extensions/wikia/ImageReview/ImageReviewImageStatus.php

```diff
@@ -108,3 +81,47 @@ function efImageReviewDisplayStatus( ImagePage $imagePage, &$html ) {
 
 	return true;
 }
+
+function fetchReviewHistory( $dbr, $cityId, $pageId ) {
+	// TODO: fetch history from ImageReview service
+
+	$res = $dbr->select(
+		'image_review_stats',
+		'*',
+		[
+			'wiki_id' => $cityId,
+			'page_id' => $pageId,
+		]
+	);
+
+	$reveiws = [];
+	while ( $row = $dbr->fetchObject( $res ) ) {
+		$data = [];
+		$data[] = User::newFromId( $row->reviewer_id )->getName();
+
+		//Possible keys:
+		// imagereview-state-0
+		// imagereview-state-1
+		// imagereview-state-2
+		// imagereview-state-3
+		// imagereview-state-4
+		// imagereview-state-5
+		$data[] = wfMessage( 'imagereview-state-' . $row->review_state )->escaped();
+		$data[] = $row->review_end . ' (UTC)';
+
+		$reviews[] = $data;
```

Comment:
@rybmat it is indeed unused as it is typoed as `$reveiws` in other places in this function ;)